cachelib/allocator/CacheAllocatorConfig.h

```diff
@@ -199,6 +201,25 @@ class CacheAllocatorConfig {
   // cachePersistence()
   CacheAllocatorConfig& usePosixForShm();
 
+  // Configures cache memory tiers. Accepts vector of MemoryTierCacheConfig.
```

Comment:
Is there a description of what Memory tiers refers to here ? There is already the notion of tiers through hybrid cache. It would help if we can point to some documentation about the memory tier here.

Reply:
I updated this method's description to list possible memory types for memory tiers. In the future, we could convert this write-up https://github.com/facebook/CacheLib/discussions/102 into a documentation section to refer to.